### PR TITLE
Replaces BOB with a Baby Space carp + Adds Carp Slippers

### PR DESCRIPTION
**Pull request by Geevies**
Her name's Ginny.

image

`ic changelog: After an industrial accident, that being a brown scaled Chief Engineer suplexing B.O.B into the crusher, Central Command has decided to give Engineering a new mascot. Ginny, the baby space carp.`

### CODE REVIEW

**Comment by Geevies** [comment]
Oh no, the chaplain rework got intertwined with this.

**Comment by Kaedwuff** [comment]
Gonna have to start over baybee.

File: aurorastation.dme [hunk 1]
```diff
@@ -843,6 +843,7 @@
 #include "code\game\objects\items\weapons\candle.dm"
 #include "code\game\objects\items\weapons\cards_ids.dm"
 #include "code\game\objects\items\weapons\cards_ids_syndicate.dm"
+#include "code\game\objects\items\weapons\chaplain_items.dm"
```
Comment (Geevies) [inline aurorastation.dme]:
```suggestion
```

**Comment by Geevies** [comment]
God bless force pushing.

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
B.O.B has suspiciously gone missing, no one appears to know where.

image
image

File: html/changelogs/geeves - carpy.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+author: Geeves
```
Comment (Alberyk) [inline html/changelogs/geeves - carpy.yml]:
```suggestion
author: Geeves, Kyres1
```

File: code/modules/mob/living/simple_animal/friendly/carp.dm [hunk 3]
```diff
@@ -0,0 +1,154 @@
+//Cat
```
Comment (Alberyk) [inline code/modules/mob/living/simple_animal/friendly/carp.dm]: Copy paste detected.

**Comment by Talkarcabbage** [comment]
what on earth is this pr

File: code/modules/mob/living/simple_animal/friendly/carp.dm (hunk 3 above)
Comment (Geevies) [reply code/modules/mob/living/simple_animal/friendly/carp.dm]: wuhwoh

**Comment by OneOneThreeEight** [comment]
geeves are u fucking ok

**Comment by KingOfThePing** [comment]
what the fuck

File: code/modules/mob/living/simple_animal/friendly/carp.dm [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+//Basic friend AI
+/mob/living/simple_animal/carp/fluff
+	var/mob/living/carbon/human/friend
+	var/befriend_job = null
+
+/mob/living/simple_animal/carp/fluff/think()
+	..()
+	if(!stat)
+		if(!buckled)
+			if (turns_since_move > 5)
```
Comment (Arrow768) [inline code/modules/mob/living/simple_animal/friendly/carp.dm]: That can be shortened into one line

File: html/changelogs/geeves - carpy.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+author: Geeves, Kyres1
+
+delete-after: True
+
+changes: 
+  - rscadd: "After an industrial accident, that being a brown scaled Chief Engineer suplexing B.O.B into the crusher, Central Command has decided to give Engineering a new mascot. Ginny, the baby space carp."
```
Comment (Arrow768) [inline html/changelogs/geeves - carpy.yml]: That´s something for the ic changelog

**Pull request merged**